src/api/sarifViewerApi.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ */
+
+import * as vscode from 'vscode';
+
+export interface OpenLogFileArguments {
+    /**
+     * Specifies options indicating whether to prompt the user when the
+     * SARIF log file shcmea is needs to be ugpraded to the latest one.
+     * If not specified, the user will be prompted. (Default is true).
+     */
+    readonly promptUserForUpgrade?: boolean;
+
+    /**
+     * Indicates whether to open the files (if not already open) and display it to the user.
+     * If not specified, the files will be displayed. (Default is true).
+     */
+    readonly openInTextEditor?: boolean;
```

Comment:
"Files" means Artifacts, right?

I assume "File" does not mean SARIF Log. But then again we are inside `OpenLogFileArguments`.

Could we disambiguate?